types/migrate-mongo/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for migrate-mongo 8.1
+// Type definitions for migrate-mongo 8.2
 // Project: https://github.com/seppevs/migrate-mongo#readme
 // Definitions by: Amit Beckenstein <https://github.com/amitbeck>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.2
+// Minimum TypeScript Version: 4.1
```

Comment:
What change to the type definitions requires version 4.1 and up?

Reply:
It’s due to the updated dependency on mongodb. I would need to check the details to be able to tell you exactly, but with typescript < 4.1, there are compile errors.

Reply:
Thanks for the clarification